Plugin Cubot/src/main/java/net/simon987/cubotplugin/Cubot.java

```diff
@@ -101,6 +103,16 @@
     private Map<Integer, HardwareModule> hardwareAddresses = new HashMap<>();
     private Map<Class<? extends HardwareModule>, Integer> hardwareModules = new HashMap<>();
 
+    /**
+    *  In game timer
+    */
+    private int time;
```

Comment:
You don't need to add more attributes to Cubot (See other comments)

Reply:
removed